fragments/master-ansible.sh

```diff
@@ -70,6 +70,15 @@ cat << EOF > $1
 EOF
 }
 
+function create_global_vars() {
+    cat << EOF > /tmp/extra_openshift_ansible_params.json
+$extra_openshift_ansible_params
+EOF
+    /usr/local/bin/merge_dict /tmp/extra_openshift_ansible_params.json \
+        /var/lib/ansible/group_vars/OSv3.yml
+    rm /tmp/extra_openshift_ansible_params.json
```

Comment:
when no params are passed and the string is empty, this is failing with:
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: + /usr/local/bin/merge_dict /tmp/extra_openshift_ansible_params.json /var/lib/ansible/group_vars/OSv3.yml
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: Traceback (most recent call last):
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: File "/usr/local/bin/merge_dict", line 51, in <module>
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: merged = load_file(sys.argv[1])
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: File "/usr/local/bin/merge_dict", line 31, in load_file
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: return json.load(open(filename))
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: File "/usr/lib64/python2.7/json/__init__.py", line 290, in load
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: **kw)
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: File "/usr/lib64/python2.7/json/__init__.py", line 338, in loads
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: return _default_decoder.decode(s)
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: File "/usr/lib64/python2.7/json/decoder.py", line 365, in decode
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: obj, end = self.raw_decode(s, idx=_w(s, 0).end())
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: File "/usr/lib64/python2.7/json/decoder.py", line 383, in raw_decode
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: raise ValueError("No JSON object could be decoded")
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: ValueError: No JSON object could be decoded
Sep 26 04:55:29 dedicated-bastion.example.com os-collect-config[9397]: [2016-09-26 04:55:29,138] (heat-config) [ERROR] Error running /var/lib/heat-config/heat-config-script/d5b7ba56-0dd1-45b
6-9fd6-693e1c572525. [1]